Migration from Self-managed Neo4j to Aura is not fully coherent on versions (#1037)
neo4j/docs-aura: modules/ROOT/pages/security/customer-managed-keys.adoc

====
The `neo4j-admin push-to-cloud` command in Neo4j v4.4 and earlier is **not** supported for instances encrypted with Customer Managed Keys.
For more information, see xref:managing-instances/backup-restore-export.adoc#neo4j-admin-upload[Upload an existing Neo4j database].

> This is because `neo4j-admin push-to-cloud` does *not* work at all with the new console and you need `neo4j-admin database upload` and version  +5.26.7 or +2025.04 

Reviewer: ```suggestion
This is because `neo4j-admin push-to-cloud` is *not* compatible with the new console.
You need to use `neo4j-admin database upload` and version  5.26.7+ or 2025.04+.
```
